src/Microsoft.ComponentDetection.Orchestrator/Services/IDetectorProcessingService.cs

```diff
@@ -8,4 +8,6 @@
 public interface IDetectorProcessingService
 {
     Task<DetectorProcessingResult> ProcessDetectorsAsync(IDetectionArguments detectionArguments, IEnumerable<IComponentDetector> detectors, DetectorRestrictions detectorRestrictions);
+
+    void SetUnusedDetectors(IEnumerable<IComponentDetector> detectors);
```

Comment:
super nit: I don't like this. Is there any way we could maybe hook into the detector restriction server service instead?

Reply:
update: added the disabled detectors to `DetectorRestrictions`